build.gradle

```diff
@@ -36,10 +36,9 @@ repositories {
 
 dependencies {
 
+	implementation 'uk.gov.laa.ccms.data:data-api:0.0.23-70d39c3-SNAPSHOT'
 
-	implementation 'uk.gov.laa.ccms.data:data-api:0.0.22-ef3e97c-SNAPSHOT'
-
-	implementation 'uk.gov.laa.ccms.soa.gateway:soa-gateway-api:0.0.38-cfdf45d-SNAPSHOT'
+	implementation 'uk.gov.laa.ccms.soa.gateway:soa-gateway-api:0.0.40-18b2f2e-SNAPSHOT'
```

Comment:
After this please can you merge the automatic prs for the latest packages for data/ebs api and soa-api.
Older version PR can be closed